infrastructure/Makefile

```
@@ -0,0 +1,12 @@
+test-infrastructure:
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)